eventmesh-webhook/eventmesh-webhook-receive/src/main/java/org/apache/eventmesh/webhook/receive/protocol/GithubProtocol.java

```diff
@@ -43,7 +43,7 @@ public class GithubProtocol implements ManufacturerProtocol {
 
     private static final String H_MAC_SHA = "HmacSHA256";
 
-    private static final char ZERO_CHAR = '0';
+    private static final Character ZERO_CHAR = '0';
```

Comment:
why you change this to Character, can this code be compiled ?

Reply:
this change is not intentional, I reverted it back